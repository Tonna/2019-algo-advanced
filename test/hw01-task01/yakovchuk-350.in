25
1 1 1 2 2 2 6 2 7 2 7 7 3 12 3 9 11 12 1 15 19 16 15 14
3
22 9
14 9
13 5
