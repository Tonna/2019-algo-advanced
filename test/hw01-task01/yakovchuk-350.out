36
20
19
